testutils/api/tenantadm.py

```diff
@@ -24,3 +26,13 @@
 
 def req_status(status):
     return {"status": status}
+
+
+class InternalApiClient(_ApiClient):
+    def __init__(self, host=HOST, scheme="http://"):
+        super().__init__(URL_INTERNAL, host, scheme)
+
+
+class ApiClient(_ApiClient):
```

Comment:
Nitpick: I think this should have an explicit Management (or Mgmt) in the class name.

Even though in `tenantadm` there is no Devices API, I think is best to be explicit. Otherwise I would be asking myself "why don't we then have an ApiClient also in all the other services?" (answer being that we would need both ManagementApiClient, DevicesApiClient bringing little value).